pkg/sawmill/sawmill.go

```diff
@@ -0,0 +1,90 @@
+package sawmill
+
+// If you're wondering why this package is named sawmill, I do too: https://clips.twitch.tv/PlacidOutstandingPelicanCharlieBitMe
+import (
+	"context"
+	"log"
+	"net/http"
+
+	"cloud.google.com/go/logging"
+	"google.golang.org/api/option"
+	logpb "google.golang.org/genproto/googleapis/logging/v2"
+)
+
+// Logger holds the newly generated error logging client, so that it can be used for our own, more simple custom logging functions instead of the ones the Google provides
+type Logger struct {
+	// The Logger
+	GoogleLogger *logging.Logger
+	// If their was an error with the initialization, it won't log to the cloud
+	InitError bool
+	// The service that logging was created for
+	ServiceName string
+}
+
+// InitClient creates a logging client
+func InitClient(projectID string, credentialsJSON string, environment string, serviceName string) (Logger, error) {
+	// The client takes the credentials in a byte array, so we do that conversion here
+	credentialsByte := []byte(credentialsJSON)
```

Comment:
We should verify that the credentialsJSON is not an empty string before trying to cast